src/content/scripting/playlist_parser_script.cc

```diff
@@ -94,9 +94,9 @@ duk_ret_t jsReadln(duk_context* ctx)
     }
 
     try {
-        auto line = self->readLine();
-        duk_push_string(ctx, line.first.c_str());
-        return line.second ? 1 : 0;
+        auto [line, trimmed] = self->readLine();
+        duk_push_string(ctx, line.c_str());
+        return trimmed;
```

Comment:
We should not implicitely convert `bool` to `duk_ret_t`

Reply:
How about now?

Reply:
I'd leave the original `? 1 : 0`. You see what you get.

Reply:
OK.